packages/db/src/export-tokens.ts

```diff
@@ -0,0 +1,24 @@
+// @ts-nocheck
```

Comment:
thanks for committing this for now, Epic Stack is doing something similar keeping loosey-goosey scripts in an `/other` dir

https://github.com/epicweb-dev/epic-stack/tree/main/other

Reply:
ah i should move it to a tmp folder!